sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5190,17 +5195,14 @@ def test_pretty_prec():
     ]
 
 
+def pprint_out(expr, *args, **kwargs):
```

Comment:
Does `pretty` lead to the same error? In that case, I'd prefer to simply use that instead of fetching the print output.

Reply:
Yes, same error.   Will push tests change to using `pretty`.